src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -185,7 +185,7 @@ Tool '{1}' (version '{2}') was successfully installed. Entry is added to the man
 * You are attempting to install a preview release and did not use the --version option to specify the version.
 * A package by this name was found, but it was not a .NET Core tool.
 * The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
-* You mistyped the name of the tool.</value>
+* More reason including PackageId naming enforcement. https://aka.ms/failure-installing-tool</value>
```

Comment:
I don't quite understand the meaning of this phrase.

Perhaps:

```suggestion
* For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool</value>
```

cc: @KathleenDollard

Reply:
Actually, I'd probably not have that as a bullet list item, just:


```xml
<value>Tool '{0}' failed to install. This failure may have been caused by:

* You are attempting to install a preview release and did not use the --version option to specify the version.
* A package by this name was found, but it was not a .NET Core tool.
* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.

or more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool</value>

Reply:
@peterhuene call it "package id naming" or just "package id"? The official term is package id

Reply:
We used "name of the tool" and "package with this name" without mentioning IDs, so I was trying to be consistent.  I will defer to whatever Kathleen or you feel is best.

Reply:
that's fair. I will go with non bullet list and "For more reasons, including package naming enforcement".

Kathleen will be back on monday, but we will be in ask mode. I'll check in first. The worst case is we have nothing to show to the user. And the current text above is pretty good

Reply:
Cool, I think we can keep the "mistyped" bullet item too since it'll be pretty common (I edited my suggestion above).

Reply:
I updated according to the suggestion above including "mistyped". Good to go?